src/Microsoft.DotNet.Interactive.FSharp/FSharpKernel.fs

```diff
@@ -373,7 +373,7 @@ type FSharpKernel () as this =
         | _ ->
             false
 
-    override _.SetVariableAsync(name: string, value: Object) : Task = 
+    override _.SetVariableAsync(name: string, value: Object, declaredType: Type) : Task = 
```

Comment:
@TIHan This is an interesting and useful change to the `SetVariableAsync` that might should work its way down to FSI. Thoughts?